app/views/blog/_date_and_author.html.erb

```diff
@@ -1,7 +1,7 @@
-<div class="blog__date">
+<div>
   <i>Published</i> <%= format_blog_date(front_matter["date"]) %>
```

Comment:
Can we get rid of the italics please? I'm trying to phase them out of the site for accessibility reasons - Gary replaced these in the design with normal text and a colon

eg Published: <name>